### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by andyxning**
This PR fixes some typos.

### CODE REVIEW

File: internal/internal.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ import (
 var (
 	// WithResolverBuilder is exported by dialoptions.go
 	WithResolverBuilder interface{} // func (resolver.Builder) grpc.DialOption
-	// WithHealthCheckFunc is not exported by dialoptions.go
+	// WithHealthCheckFunc is exported by dialoptions.go
 	WithHealthCheckFunc interface{} // func (HealthChecker) DialOption
```
[the remark this answers is not in the document]
Comment (easwars) [reply internal/internal.go]: Just realized that the `exported` in these comments don't mean the same thing as a Go exported variable. Would you mind changing them to `set` instead. Thank you.

[the remark this answers is not in the document]
Comment (andyxning) [reply internal/internal.go]: Done

[the remark this answers is not in the document]
Comment (andyxning) [reply internal/internal.go]: @easwars PTAL.

**Comment by easwars** [comment]
Thank you for your contribution.

**Pull request merged**